src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/BlazorDebugConfigurationProvider.ts

```diff
@@ -31,33 +26,32 @@ export class BlazorDebugConfigurationProvider implements vscode.DebugConfigurati
         };
         const browser = {
             name: '.NET Core Debug Blazor Web Assembly in Browser',
-            type: configuration.browser === 'edge' ? 'edge' : 'pwa-chrome',
+            type: configuration.browser === 'edge' ? 'pwa-edge' : 'pwa-chrome',

```

Comment:
It's `pwa-msedge` rather than `pwa-edge`--to match the existing Chromium-Edge debug type 🙂

Reply:
Good catch -- fixed!